tx-parser/src/lib.rs

```diff
@@ -13,12 +13,17 @@
 extern crate edn;
 extern crate combine;
 extern crate mentat_tx;
+extern crate mentat_parser_utils;
+
+#[macro_use]
+extern crate mentat_query_parser;
 
 use combine::{any, eof, many, optional, parser, satisfy_map, token, Parser, ParseResult, Stream};
 use combine::combinator::{Expected, FnParser};
 use edn::symbols::NamespacedKeyword;
 use edn::types::Value;
-use mentat_tx::entities::*;
+use mentat_tx::entities::{Entid, EntidOrLookupRef, Entity, LookupRef, ValueOrLookupRef};
```

Comment:
I changed it because I misread an error message telling me it wasn't needed (it was actually the one down around line 274 in `mod tests`).

I agree with @rnewman's logic, and I'd also add that some specificity about what's required and what isn't is a good thing.